src/InteractiveTutorial.ts

```diff
@@ -104,7 +104,10 @@ function iTutorialEnd(): void {
   ITutorial.isRunning = false;
   ITutorial.currStep = iTutorialSteps.Start;
   const messages = Player.getHomeComputer().messages;
-  const handbook = LiteratureNames.HackersStartingHandbook;
+  const handbook = LiteratureName.HackersStartingHandbook;
+  if (!isFilePath(handbook) || !isAbsolutePath(handbook)) {
```

Comment:
Why do you need this check here (and places like here)? You're dealing with a constant.

Reply:
This is to provide a runtime error in case someone changes something that makes that constant no longer obey the rules of the type it's supposed to be.

Reply:
I went with just doing the strict typecheck in Literature, Message, and Program constructors, which are also forced to process every enum member due to the types for `Literatures`, `Messages`, and `Programs`.

Then I loosened up the type a little on servers.messages and servers.programs so the hardcoded filenames could be used directly without extra typechecking them every time.